Sources/SQLKit/Expressions/Queries/SQLDropEnum.swift

```diff
@@ -1,39 +1,46 @@
-/// `DROP TYPE` query.
+/// An expression representing a `DROP TYPE` query. Used to delete enumeration types.
 ///
+/// ```sql
+/// DROP TYPE IF EXISTS "enum_type" CASCADE:
+/// ```
+/// 
+/// This expression does _not_ check whether the current dialect supports separate enumeration types; users should
+/// take care not to use it with incompatible drivers.
+/// 
 /// See ``SQLDropEnumBuilder``.
 public struct SQLDropEnum: SQLExpression {
-    /// Type to drop.
-    public let name: any SQLExpression
+    /// The name of the type to drop.
+    public var name: any SQLExpression
 
-    /// The optional `IF EXISTS` clause suppresses the error that would normally
-    /// result if the type does not exist.
+    /// If `true`, requests idempotent behavior (e.g. that no error be raised if the named type does not exist).
+    ///
+    /// Ignored if not supported by the dialect.
     public var ifExists: Bool
 
-    /// The optional `CASCADE` clause drops other objects that depend on this type
-    /// (such as table columns, functions, and operators), and in turn all objects
-    /// that depend on those objects.
-    public var cascade: Bool
+    /// A drop behavior.
+    ///
+    /// Ignored if not supported by the dialect. See ``SQLDropBehavior``.
+    public var dropBehavior: SQLDropBehavior
```

Comment:
We happy with this break?

Reply:
There is no breakage. No memberwise initializer is public (or exists, for that matter), and the default value of the additional property has the same effect as when it didn't exist.